lib/diagnostic.rb

```diff
@@ -43,6 +47,14 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    lines = File.open(ARGV[0]) do |file|
+      file.map(&:itself)
```

Comment:
`map(&:itself)` just creates a duplicate array, with the same content as the original.
Instead of doing this, we want to get rid of the `\n` at the end of each line and (except for the final one).